src/main/webapp/cdn/blockly/generators/propc/base.js

```diff
@@ -176,9 +176,9 @@ Blockly.Blocks.math_number = {
             }
             if (rangeVals[0] === 'R' && (range[2] < range[0] || range[2] > range[1]) && Math.abs(range[0] - range[1]) <= 10000000) {
                 if (this.getField('TITLE')) {
-                    if (range[1] >= 2147483648) {
```

Comment:
Going forward, let's use constants in place of the numeric value to make the code more readable. Using something like MAX_INT_VALUE makes it clear what that number represents.